docs/source/release_notes/release-notes.rst

```diff
@@ -52,6 +70,7 @@ develop
 
     *    - |fixed|
          - KVS migration no longer fails when the old ``_scrub`` table is present.
+           This allows users who have not performed the `Scrub migration <https://github.com/palantir/atlasdb/pull/2232>`__ to perform KVS migrations.
```

Comment:
We really should have scrub migration docs, but that's a PR. Not sure I'd bother linking to the PR.

"This unblocks KVS migrations for users who have data in ``_scrub`` but have not migrated from ``_scrub`` to ``_scrub2``." ?

Reply:
Yep, that makes sense. I guess for someone unfamiliar with the change, that PR is not going to be particularly helpful. I'll change this